styx-common/src/main/java/com/spotify/styx/api/Middlewares.java

```diff
@@ -159,17 +159,18 @@ private static GoogleIdToken verifyIdToken(String s) {
   public static <T> Middleware<AsyncHandler<Response<T>>, AsyncHandler<Response<T>>> auditLogger() {
     return innerHandler -> requestContext -> {
       final Request request = requestContext.request();
-      final AuthContext authContext = auth(requestContext);
 
       if (!"GET".equals(request.method())) {
         LOG.info("[AUDIT] {} {} by {} with headers {} parameters {} and payload {}",
-            request.method(),
-            request.uri(),
-            authContext.user().map(idToken -> idToken.getPayload().getEmail()).orElse("anonymous"),
-            filterHeaders(request.headers()),
-            request.parameters(),
-            request.payload().map(ByteString::utf8).orElse("")
-                .replaceAll("\n", " "));
+                 request.method(),
+                 request.uri(),
+                 auth(requestContext).user().map(idToken -> idToken.getPayload()
```

Comment:
Not sure of the whole story. I thought that middleware shall be used if the actual handler needs to access `user`, and so far we don't have that case.